### ISSUE TITLE
Not an issue: a statement

### ISSUE THREAD

**Issue by HeavilyModdedEnder3** [SnakeOilXY/klipper-extended-corexy-kinematic#2]
Looks like the new Ratrig design. Did you submit it to Klipper yet?

**Comment by ChipCE** [comment]
Yeah, it is quite similar to the new Ratrig kinematic. Work well on my machine for years, but I think the code is not in the state to be able to be accepted by klipper main branch.
